src/plugins/fs/generic.c

```diff
@@ -50,6 +50,76 @@ typedef enum {
     BD_FS_GET_FREE_SPACE,
 } BDFsOpType;
 
+static const BDFSFeatures fs_features[BD_FS_LAST_FS] = {
+    { 0 }, { 0 },
```

Comment:
It would be nice to have the explanation of the two `{ 0 }`s here.

Reply:
Ok, I have added a comment for that.